.github/build_docs.py

```diff
@@ -0,0 +1,59 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I recommend using the BTD Action. The build procedure in VUnit/vunit is particular, but here we can use the Action as-is.